doc/functions/date.rst

```diff
@@ -43,6 +43,9 @@ If no argument is passed, the function returns the current date:
         $twig = new Twig_Environment($loader);
         $twig->getExtension('core')->setTimezone('Europe/Paris');
 
+        // as of Twig 1.26
```

Comment:
should be moved before the legacy way IMO (and to be consistent with the symfony documentation standards, the deprecated way should be removed entirely)